### PR TITLE
Audience Manager Mixin

### PR DESCRIPTION
**Pull request by ankushkd**
This PR is for addition of Audience Manager mixin, consisting of various XDMs that are required for audience manager experience event.
This can be used to create Audience Manager Experience Event schema and will also host any future custom required fields.

### CODE REVIEW

**Comment by ankushkd** [comment]
@kstreeter can yiu please review this and let me know if any changes are required?

**Comment by kstreeter** [comment]
Hey @cdegroot-adobe let me know if you are good with this and we will merge.

File: extensions/adobe/experience/aam-experienceevent.schema.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Audience Manager Template Mixin",
+  "type": "object",
+  "description": "Adobe Audience Manager Mixin for use with Schemas for Solution data ingestion. Includes the core/standard ExperienceEvent as well as the other required Core mixins.",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/context/experienceevent-segmentmembership",
+    "https://ns.adobe.com/xdm/context/experienceevent-enduserids-deprecated",
```
Comment (cdegroot-adobe) [inline extensions/adobe/experience/aam-experienceevent.schema.json]: Why do we still need the deprecated user ids? At this point we should be not using them.

File: extensions/adobe/experience/aam-experienceevent.schema.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    "https://ns.adobe.com/xdm/context/device",
+    "https://ns.adobe.com/xdm/context/environment",
+    "https://ns.adobe.com/xdm/context/placecontext",
+    "https://ns.adobe.com/xdm/context/profileStitch"
```
Comment (cdegroot-adobe) [inline extensions/adobe/experience/aam-experienceevent.schema.json]: These last 4 includes are all data types, they do not introduce concrete fields.
You should include the equivalent https://ns.adobe.com/xdm/context/experienceevent-*

File: extensions/adobe/experience/aam-profile.schema.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Audience Manager Profile Template Mixin",
+  "type": "object",
+  "description": "Adobe Audience Manager Mixin for use with Schemas for Solution data ingestion. Includes the core/standard Profile XDM as well as the other required Core mixins.",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/context/profile-segmentation",
+    "https://ns.adobe.com/xdm/context/profile-identities-deprecated",
```
Comment (cdegroot-adobe) [inline extensions/adobe/experience/aam-profile.schema.json]: As above, can we stop using this now?

File: extensions/adobe/experience/aam-profile.schema.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    "https://ns.adobe.com/xdm/context/device"
```
Comment (cdegroot-adobe) [inline extensions/adobe/experience/aam-profile.schema.json]: Why are you adding device information to a profile? It was designed for an instance observation?

File: extensions/adobe/experience/aam-experienceevent.schema.json (hunk 1 above)
Comment (ankushkd) [reply extensions/adobe/experience/aam-experienceevent.schema.json]: Removed it.

File: extensions/adobe/experience/aam-experienceevent.schema.json (hunk 2 above)
Comment (ankushkd) [reply extensions/adobe/experience/aam-experienceevent.schema.json]: Changed.

File: extensions/adobe/experience/aam-profile.schema.json (hunk 3 above)
Comment (ankushkd) [reply extensions/adobe/experience/aam-profile.schema.json]: Removed.

**Comment by ankushkd** [comment]
Thanks for reviewing and comment @cdegroot-adobe 

> We use this pattern for other cases where data connectors create schemas for experience event, and the *-all mixin is a simple controller ease creation of the schemas in the tenants and also to add consistency. Is that the idea here?
Yes. This will be used for creating schema via solution connector. I would be able to add *-all (with solution specific fields) later into this right?

**Comment by kstreeter** [comment]
@cdegroot-adobe, this is good?

**Pull request merged**